Zend/tests/data_classes/assign_op.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Assign op on data classes
+--FILE--
+<?php
+
+data class Box {
```

Comment:
Could we rather call them `immutable` classes? IMO this would better express the actual behavior than `data`.

Reply:
They're not immutable though, unless you'd consider arrays immutable?

Reply:
We could consider `value class`, or `struct`. Let's discuss this on the list though. :)

Reply:
Ah no, sorry, I only had a quick look and slightly misunderstood what this does... wishful thinking I guess